packages/widget_toolkit_biometrics/README.md

```diff
@@ -0,0 +1,154 @@
+![CI][ci_badge_lnk] [![codecov][codecov_badge_lnk]][codecov_branch_lnk] ![style][code_style_lnk] ![license][license_lnk]  
```

Comment:
Please add the package to the main README.md file as well
https://github.com/Prime-Holding/widget_toolkit/tree/feature/new-package-widget-toolkit-biometrics